compose/foundation/foundation/src/commonMain/kotlin/androidx/compose/foundation/text2/input/internal/TextFieldKeyEventHandler.kt

```diff
@@ -76,10 +76,6 @@ internal abstract class TextFieldKeyEventHandler {
         singleLine: Boolean,
         onSubmit: () -> Unit
     ): Boolean {
-        if (event.type != KeyEventType.KeyDown) {
```

Comment:
> On the desktop, typed events are not KeyDown events

Since the changes in common:
1. Will it still work on Android?
2. Should we fix it in AOSP first?

Reply:
1. I think so.
2. I'm talking to Zach about it.

Reply:
Let's slowly change our process to make a fix in AOSP first, and then cherry-pick it after merging to `androidx-main`.

And merge commonMain changes only if they are critical and/or we need them soon.

Not sure about this fix. There are 2 factors:
- we need to merge it before 1.6.10-rc01
- the `androidx-main` code might be changed a lot right now and isn't worth upstreaming. So we just fix a small bug for just 1.6.10

Reply:
This part of the code hasn't changed in `androidx-main` tip, so it's a very easy upstreaming.